services-geojson/src/commonMain/kotlin/org/maplibre/geojson/model/GeometryCollection.kt

```diff
@@ -0,0 +1,54 @@
+package org.maplibre.geojson.model
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.encodeToString
+import org.maplibre.geojson.utils.json
+import kotlin.jvm.JvmStatic
+
+/**
+ *
+ */
+@Serializable
+@SerialName("GeometryCollection")
+data class GeometryCollection(
+    /**
+     *
+     */
+    val geometries: List<Geometry>,
+
+    /**
+     *
+     */
+    @SerialName("bbox")
+    override val boundingBox: BoundingBox?,
```

Comment:
```suggestion
    override val boundingBox: BoundingBox? = null,
```
?

(This might also affect serialization, as default parameters are by default not serialized explicitly `"bbox": null` when the value is equal to the default)

Reply:
We are encoding default values, but skip fields with `NULL` values. 


```kotlin
    // org.maplibre.geojson.utils.Json.kt
    
    encodeDefaults = true
    explicitNulls = false
```

> A GeoJSON object MAY have a "bbox" member

So we can left this field, with no problems.